dataactbroker/handlers/fileHandler.py

```diff
@@ -1262,73 +1262,76 @@ def list_submissions(page, limit, certified, sort='modified', order='desc'):
 
     offset = limit * (page - 1)
 
+    submission_columns = [Submission.submission_id, Submission.cgac_code, Submission.user_id,
+                          Submission.publish_status_id, Submission.d2_submission, Submission.number_of_warnings,
+                          Submission.number_of_errors, Submission.updated_at, Submission.reporting_start_date,
+                          Submission.reporting_end_date]
+
+    cgac_columns = [CGAC.cgac_code, CGAC.agency_name]
+    user_columns = [User.user_id, User.name]
+
+    columns_to_query = submission_columns + cgac_columns + user_columns
+
     cgac_codes = [aff.cgac.cgac_code for aff in g.user.affiliations]
-    query = sess.query(Submission).filter_by(d2_submission=False)
+    query = sess.query(*columns_to_query).\
+        outerjoin(User, Submission.user_id == User.user_id). \
+        outerjoin(CGAC, Submission.cgac_code == CGAC.cgac_code).\
+        filter(Submission.d2_submission.is_(False))
     if not g.user.website_admin:
         query = query.filter(sa.or_(Submission.cgac_code.in_(cgac_codes),
                                     Submission.user_id == g.user.user_id))
     if certified != 'mixed':
         if certified == 'true':
-            query = query.filter(Submission.publish_status_id == PUBLISH_STATUS_DICT['published'])
+            query = query.filter(Submission.publish_status_id != PUBLISH_STATUS_DICT['unpublished'])
         else:
-            query = query.filter(Submission.publish_status_id != PUBLISH_STATUS_DICT['published'])
+            query = query.filter(Submission.publish_status_id == PUBLISH_STATUS_DICT['unpublished'])
+
+    print(query)
 
-    arr = [serialize_submission(s) for s in query]
+    total_submissions = query.count()
 
     options = {
-        'modified': 'last_modified',
-        'reporting': 'reporting_start_date',
-        'status': 'status',
-        'agency': 'agency'
+        'modified': {'model': Submission, 'col': 'updated_at'},
+        'reporting': {'model': Submission, 'col': 'reporting_start_date'},
+        # 'status': 'status',
```

Comment:
We removing this sort? If so, do we really need this commented out still? Could just delete it.